v5_MIGRATION.md

```diff
@@ -0,0 +1,54 @@
+# Android Popup Bridge v5 (Beta) Migration Guide
+
+See the [CHANGELOG](/CHANGELOG.md) for a complete list of changes. This migration guide outlines the basics for updating your popup bridge integration from v4 to v5.
+
+> When migrating from v3 to v5, make sure to view the [/v4_MIGRATION.md] first.
+
+## Setup
+
+First, update the popup bridge dependency version in your `build.gradle` file:
+
+```groovy
+dependencies {
+  implementation 'com.braintreepayments.api:popup-bridge:5.0.0'
+}
+```
+
+## Usage
+
+The primary difference betwen v4 and v5 is the host activity no longer has to call `deliverPopupBridgeResult()` in `onResume()`. In v5, this is done internally with an Android Jetpack [Lifecycle Observer](https://developer.android.com/topic/libraries/architecture/lifecycle).
+
+To use PopupBridge, instantiate a `PopupBridgeClient`:
+
+```java
```

Comment:
I've been enjoying using the `diff` markdown syntax to show lines that will get added or deleted in a merchant integration ([see example](https://github.com/braintree/braintree_ios/blob/3ds-client/V6_MIGRATION.md#3d-secure)). Might be nice to use that here!